### DISCUSSION TITLE
Using different atom_style in dpgen

### DISCUSSION THREAD

**Discussion by Cheng-JW** [deepmodeling/dpgen#1429] [Q&A]
I am trying to use dpgen with a GCMC simulation to generate data. However, I noticed that all confs generated by dpgen is in atom_style: atomic, which I want to convert into full style for GCMC. Is it possible in dpgen or do I need to change the codes?

**Comment by njzjz** [comment]
This is something that should be implemented in dpdata:

https://github.com/deepmodeling/dpdata/blob/master/dpdata/lammps/lmp.py
